### PR TITLE
UUID as point id

### PR DESCRIPTION
**Pull request by generall**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally using ``cargo fmt`` command prior to submission?
3. [x] Have you checked your code using ```cargo clippy``` command?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by generall** [comment]
https://github.com/qdrant/qdrant/issues/255

**Comment by generall** [comment]
It might seems like a lot of changs, but actually it is mostly an additional `.into()` for each const `point_id` used in tests

File: lib/collection/src/collection.rs [hunk 1]
```diff
@@ -139,34 +137,39 @@ impl Collection {
             });
         }
 
+        // Retrieve 1 extra point to determine proper `next_page_offset`
```
Comment (agourlay) [inline lib/collection/src/collection.rs:140]: Is this an unrelated issue existing in the main branch?

Comment (generall) [reply lib/collection/src/collection.rs:140]: no, it's due to the change, that we can not calculate the next ID anymore

Comment (agourlay) [reply lib/collection/src/collection.rs:140]: got it thanks :+1:

**Comment by agourlay** [comment]
All those `into()` are rather noisy but I do not see a better alternative.

**Comment by generall** [comment]
> All those into() are rather noisy but I do not see a better alternative.

same, there are no implicit conversion like in Scala. But it is only noisy in tests

**Review by agourlay — APPROVED** [review]

**Pull request merged**